ui/src/tool-panel/km-post/km-post-infobox.tsx

```diff
@@ -191,15 +191,15 @@ const KmPostInfobox: React.FC<KmPostInfoboxProps> = ({
                     <InfoboxFieldWithEditLink
                         qaId="km-post-gk-coordinate-system"
                         label={t('tool-panel.km-post.layout.gk-coordinates.coordinate-system')}
-                        value={`${gkCoordinateSystem?.name} ${gkCoordinateSystem?.srid}`}
+                        value={`${gkCoordinateSystem?.name ?? '-'} ${gkCoordinateSystem?.srid ?? ''}`}
```

Comment:
Tämän saisi ehkä helpommin luettavaan muotoon tarkastelemalla koko `gkCoordinateSystem`:in olemassaolon ternaryllä, palauttamalla wanhantyylisen stringin jos se on ja `'-'` mikäli se on `undefined`. Toki johtaa vähän pidempään riviin, mutta tässä nykyisessä mallissa rupesin ensimmäisenä miettimään että mitä tapahtuu jos `gkCoordinateSystem` on määritelty mutta sen `.name` on `undefined`, mutta `.srid` saa arvon. Toki tyypityshän tämän estää (molemmilla on pakko olla arvo,) mutta koska tuo tyyppimääritys ei ollut mulla suoraan muistissa niin tämä asia piti käydä tarkastamassa

Reply:
Njoo, tavallaan. Toisaalta onko se yhtään parempi jos noin olisikin? Jos olisi sallittua löytyä srid mutta ei nimeä niin eikös se ole parempi että se näytetään, vaikka nimi jäisikin -:ksi? Ehkä tuolle voisi tehdä jonkun formatointifunkkarin ainakin, koska oikeastaan sen haluttanee olevan aika samanmuotoinen kaikkialla.

Reply:
Enpäs muistanut alkuperäistä kommenttia kirjoittaessa mutta nyt tuli mieleen, meillähän on jo muuten olemassa `<CoordinateSystemView>`-komponentti joka hoitaa tuon näyttämisen. Se tosin todennäköisesti hoitaa hommansa samalla tavalla väärin kuin miten tämä hoiti aiemmin, koska koodi näyttää identtiseltä (eli näyttää `undefined undefined`.) Tämän voisi vaihtaa siihen, ja nämä korjaukset voisi tehdä sinne

Reply:
Joo näyttäisi että tuo komponentti ottaa non-null coordinatesystemin sisään. Pistän sen käyttöön, jolloin se menee ihan normaalilla `cs && <CoordinateSystemView...` tyylillä. Funktioin silti tuon tyylittelyn koska sitä käytetään muuallakin (dropdownit) ja silloin funkkari voi ottaa sisään non-nullia ja tuo mainitsemasi koodin outous poistuu sitä kautta.

Reply:
Hups, eikäs otakaan kun nullablen. Korjataan siis sekin...